docs/topics/ephemeral-os.md

```diff
@@ -0,0 +1,35 @@
+# Ephemeral OS
+
+This document describes how to use ephemeral OS for VMs provisioned in Azure. Ephemeral OS uses local VM storage for changes to the OS disk. Storage devices local to the VM host will not be bound by normal managed disk SKU limits. Instead they will always be capable of saturating the VM level limits. This can significantly performance performance on the OS disk. Ephemeral storage used for the OS will not persist between maintenance events and VM redeployments. This is ideal for stateless base OS disks, where any stateful data is kept elsewhere. 
```

Comment:
```suggestion
This document describes how to use ephemeral OS for VMs provisioned in Azure. Ephemeral OS uses local VM storage for changes to the OS disk. Storage devices local to the VM host will not be bound by normal managed disk SKU limits. Instead they will always be capable of saturating the VM level limits. This can significantly increase performance of the OS disk. Ephemeral storage used for the OS will not persist between maintenance events and VM redeployments. This is ideal for stateless base OS disks, where any stateful data is kept elsewhere. 
```

Reply:
"This can significantly performance performance" 😄

Reply:
You are unambiguous regarding the key point.